Dockerfiles/backend.Dockerfile

```diff
@@ -40,5 +48,9 @@ ENV GUNICORN_CMD_ARGS "--bind=0.0.0.0:5000 --workers=2 --thread=4 --worker-class
 # Set working directory - 'code' dir in container, 'code' dir locally (in code)
 WORKDIR /code/dds_web
 
+# Get the built frontend
+RUN mkdir ./static/css
+COPY --from=nodebuilder /build/css ./static/css
```

Comment:
There may be javascript files as well from npm? eg. datatables..

Reply:
These ones I'm thinking of:

https://github.com/ScilifelabDataCentre/dds_web/blob/973e1dd16a08fc21a53137df9789fc2152008be2/dds_web/templates/base.html#L68-L71

Reply:
Might be nicer to copy these within the `npm run` commands into `static` instead of linking directly into the `node_modules` directory.. I guess we don't want that entire thing on the production site? (though shouldn't do much _harm_ I guess..)

Reply:
Solved for now by copying everything from the static folder, including `node_modules`.